event_types.go

```diff
@@ -161,13 +161,21 @@ type IssueEvent struct {
 		Username  string `json:"username"`
 		AvatarURL string `json:"avatar_url"`
 	} `json:"assignee"`
-	Labels []Label `json:"labels"`
+	Assignees []struct {
+		Name      string `json:"name"`
+		Username  string `json:"username"`
+		AvatarURL string `json:"avatar_url"`
+	} `json:"assignees"`
+	Labels  []Label `json:"labels"`
 	Changes struct {
 		Labels struct {
 			Previous []Label `json:"previous"`
 			Current  []Label `json:"current"`
 		} `json:"labels"`
-		UpdatedByID []int `json:"updated_by_id"`
+		UpdatedByID struct {
+			Previous []Label `json:"previous"`
+			Current  []Label `json:"current"`
+		} `json:"updated_by_id"`
```

Comment:
This field was updated a few weeks ago in PR #593 to match how Gitlab uses this field. I'm guessing this is something that recently changed in the Gitlab API, so if you are using an older Gitlab version you may want to keep using v0.16.1 of this package instead of `master`?

Reply:
Hi all, 

In the PR #593 @itscaro  you say you have read it in the documentation. I don't found it can you share a link please ?

I'm use master gitlab (so a really fresh version), and when I run a webhook I have this json :

```json
{
   "object_kind":"issue",
   "event_type":"issue",
   "user":{
      "name":"manland",
      "username":"manland",
      "avatar_url":"https://www.gravatar.com/avatar/c6b552a4cd47f7cf1701ea5b650cd2e3?s=80\\u0026d=identicon"
   },
   "project":{
      "id":24,
      "name":"webhook",
      "description":"",
      "web_url":"http://localhost:3000/manland/webhook",
      "avatar_url":null,
      "git_ssh_url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
      "git_http_url":"http://localhost:3000/manland/webhook.git",
      "namespace":"manland",
      "visibility_level":20,
      "path_with_namespace":"manland/webhook",
      "default_branch":"master",
      "ci_config_path":null,
      "homepage":"http://localhost:3000/manland/webhook",
      "url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
      "ssh_url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
      "http_url":"http://localhost:3000/manland/webhook.git"
   },
   "object_attributes":{
      "author_id":50,
      "closed_at":null,
      "confidential":false,
      "created_at":"2019-04-06 21:20:45 UTC",
      "description":"test label issue",
      "due_date":null,
      "id":183,
      "iid":3,
      "last_edited_at":null,
      "last_edited_by_id":null,
      "milestone_id":null,
      "moved_to_id":null,
      "project_id":24,
      "relative_position":1073743323,
      "state":"opened",
      "time_estimate":0,
      "title":"Test label issue",
      "updated_at":"2019-04-06 21:21:08 UTC",
      "updated_by_id":50,
      "url":"http://localhost:3000/manland/webhook/issues/3",
      "total_time_spent":0,
      "human_total_time_spent":null,
      "human_time_estimate":null,
      "assignee_ids":[

      ],
      "assignee_id":null,
      "action":"update"
   },
   "labels":[
      {
         "id":36,
         "title":"violet",
         "color":"#A295D6",
         "project_id":24,
         "created_at":"2019-04-06 21:21:06 UTC",
         "updated_at":"2019-04-06 21:21:06 UTC",
         "template":false,
         "description":null,
         "type":"ProjectLabel",
         "group_id":null
      }
   ],
   "changes":{
      "updated_at":{
         "previous":"2019-04-06 21:20:45 UTC",
         "current":"2019-04-06 21:21:08 UTC"
      },
      "updated_by_id":{
         "previous":null,
         "current":50
      },
      "labels":{
         "previous":[

         ],
         "current":[
            {
               "id":36,
               "title":"violet",
               "color":"#A295D6",
               "project_id":24,
               "created_at":"2019-04-06 21:21:06 UTC",
               "updated_at":"2019-04-06 21:21:06 UTC",
               "template":false,
               "description":null,
               "type":"ProjectLabel",
               "group_id":null
            }
         ]
      }
   },
   "repository":{
      "name":"webhook",
      "url":"ssh://rmaneschi@localhost:2222/manland/webhook.git",
      "description":"",
      "homepage":"http://localhost:3000/manland/webhook"
   }
}
```

With changes -> updated_by_id{previous, current} like other fields.

Anyway i'm sorry because i have done this PR too quickly. It's not labels but int :sweat_smile: In my app i don't use this field it's just to pass the deserialization, meaculpa :man_facepalming:

Reply:
In the documentation which Links is in the doc of the fonction, you can find thaïs

```
  "changes": {
    "updated_by_id": [null, 1],
```

Reply:
@manland do you have an update on this one? Please see the request body example here that shows this format: https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#merge-request-events

Are you by any change testing against an older Gitlab instance!

Reply:
@svanharmelen i have tested with latest version and the documentation appears to be false... I need to double check and to make a MR if its true!

Reply:
Check... Please let me know the outcome of your investigation. Thanks!